### ISSUE TITLE
Screen reader focus does not sync with keyboard focus on color pickers sliders present under ‘Color’ dialog box’.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#136]
User Experience:

Cognitive impaired users and screen reader users will face difficulty if screen reader focus is not syncing with keyboard focus, users won't be able to perform functionality over such control.

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode

4. Navigate to the Left nav and select "Styles" button.

5. Navigate to style page and select Edit Color  button.

6. Enable screen reader and navigate to the various controls of the "Edit Color" dialog box

7. Verify the issue.

 

Actual Result:

Screen reader focus does not sync with keyboard focus on color pickers sliders present under ‘Color’ dialog box’.

 

Expected result:

Screen reader focus should be sync with keyboard focus on color pickers sliders present under ‘Color’ dialog box’.

 

Note:

Similar issue exist with Chrome + JAWS and Chrome + NVDA

 

MAS Reference:

MAS 4.2.10 - Tracking of Focus and Selection Attributes (33J)

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.
